packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from 'terra-section-header';
+import Toggle from 'terra-toggle';
+
+const VALID_LEVELS = [1, 2, 3, 4, 5, 6];
+
+const propTypes = {
+  /**
+   * Content in the body of the toggle section header component that will be expanded or hidden.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text to be displayed in the section header.
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, or 6.
+   */
+  level: PropTypes.oneOf(VALID_LEVELS),
+  /**
+   * Callback function triggered when toggle-section-header is closed.
+   */
+  onClose: PropTypes.func,
+  /**
+   * Callback function triggered when toggle-section-header is opened.
+   */
+  onOpen: PropTypes.func,
+  /**
+   * Used to set props and HTML attributes on the underlying section-header.
+   */
+  sectionHeaderAttrs: PropTypes.object,
+  /**
+   * Sets the toggle-section-header to be animated when it is opened or closed.
+   */
+  isAnimated: PropTypes.bool,
+  /**
+   * Sets the toggle-section-header initial state to open.
+   */
+  isInitiallyOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  level: 2,
+  sectionHeaderAttrs: {},
+};
```

Comment:
Hmm.. by not providing defaultProps entries for optionals above, the props table fills out kinda.. weird
<img width="745" alt="screen shot 2018-03-12 at 5 38 21 pm" src="https://user-images.githubusercontent.com/34115417/37313216-3af18bda-261c-11e8-8d8e-c2a531af5e19.png">

most notably, the booleans:
>isAnimated defaults to none? isInitiallyOpen defaults to none?

Reply:
wondering if this is an issue with the props table?

Reply:
That would be an issue with the props table

Reply:
Let's default `isAnimated` `isInitiallyOpen` to false. That's what we did for the toggle button component: http://engineering.cerner.com/terra-core/#/site/toggle-button

Reply:
done 6407343afb11deba8978d49480abb921f14c16a8